.travis.yml

```diff
@@ -7,106 +7,157 @@ branches:
     - main
 cache:
   directories:
-    - $HOME/$TRAVIS_BUILD_ID
-    - $HOME/docker
-env:
-  global:
-    - DOCKER_IMAGE='cerner/terra-core'
-    - TAG="${TRAVIS_PULL_REQUEST_SHA:=latest}"
+    - travis-build
 before_install:
-  - sudo chown -R travis:travis $HOME/docker
-  # Load cached docker images
-  - if [[ -d $HOME/docker ]]; then ls $HOME/docker/*.tar.gz | xargs -I {file} sh -c "zcat {file} | docker load"; fi
-before_cache:
-  - sudo chown -R travis:travis $HOME/docker
-  - if [ $TRAVIS_TEST_RESULT == 1 ]; then rm -f $HOME/docker/*.tar.gz; fi
+  # Clean install on any new build
+  - npm run clean
 jobs:
   include:
-    - stage: build
-      before_install: true
-      script:
-        - sudo chown -R travis:travis $HOME/docker
-        - 'rm -f $HOME/docker/*.tar.gz'
-        - travis_wait docker-compose build test-ci
-        - travis_wait docker-compose --verbose pull standalone-chrome
-        - >
-          mkdir -p $HOME/docker && docker images -a --filter='dangling=false' --format '{{.Repository}}:{{.Tag}} {{.ID}}'
-          | xargs -n 2 -t sh -c 'test -e $HOME/docker/$1.tar.gz || docker save $0 | gzip -2 > $HOME/docker/$1.tar.gz'
-    - stage: danger lint and jest
-      script: travis_wait docker-compose run -e DANGER_GITHUB_API_TOKEN=$DANGER_GITHUB_API_TOKEN -e HAS_JOSH_K_SEAL_OF_APPROVAL=$HAS_JOSH_K_SEAL_OF_APPROVAL -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG test-ci sh -c 'npm run danger && npm run lint && npm run jest'
-    - stage: compile - default theme
-      script: travis_wait docker-compose run test-ci npm run compile:prod -- --env.disableAggregateThemes
+    - stage: danger lint jest and compile
+      script:
+        # clear the cache of any old build artifacts
+        - rm -rf ./travis-build
+        - npm run danger
+        - npm run lint
+        - npm run jest
+        # These can't be built in parallel because the cache is a single object and you can't add to it in parallel
```

Comment:
Can we run all 3 of these in parallel with danger lint and jest?

Reply:
Never mind. I think I understand your comment now.

Reply:
Yeah at one point i had all three compile jobs running in parallel but only the last to complete would be cached.

We could get a couple of minutes back by deprecating terra-markdown and terra-doc-template (the only remaining   components that use theme aggregation)